spring-cloud-stream-binders/spring-cloud-stream-binder-test/src/main/java/org/springframework/cloud/stream/binder/BrokerBinderTests.java

```diff
@@ -74,9 +74,9 @@ public void handleMessage(Message<?> message) throws MessagingException {
 		assertEquals(2, count.get());
 		assertNull(spyOn("direct.0").receive(true));
 
-		// Remove direct binding and bind producer to the bus
-		bus.unbindConsumers("direct.0");
-		busBindUnbindLatency();
+		// Remove direct binding and bind producer to the binder
```

Comment:
this comment no longer makes sense IMO; nothing is bound to the binder, it's bound BY the binder